### PR TITLE
[CI]Run xwfm-test on master

### PR DESCRIPTION
**Pull request by tmdzk**
## Summary

Run xwfm-test on master 

## Test Plan

Let CI run, maje sure xwfm workflow has been created and xwfm-test ran successfully

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by 119Vik** [comment]
Can one of the admins verify this patch?

File: .circleci/config.yml [hunk 1]
```diff
@@ -764,9 +764,42 @@ jobs:
...
+            echo export MAGMA_ROOT=$(pwd) >> $BASH_ENV
+            echo export AWS_ACCESS_KEY_ID="$(printenv XWF_AWS_ACCESS_KEY_ID)" >> $BASH_ENV
+            echo export AWS_SECRET_ACCESS_KEY="$(printenv XWF_AWS_SECRET_ACCESS_KEY)" >> $BASH_ENV
+      - run:
+          name: "Set Python Version"
+          command: pyenv global 3.7.0
+      - run:
+          command: |
+            env
+            docker login -u ${XWF_ARTIFACTORY_USER} -p ${XWF_ARTIFACTORY_API_KEY} facebookconnectivity-openflow-xwfm-docker.jfrog.io
```
Comment (xjtian) [inline .circleci/config.yml]: put the docker registry in an env var

File: .circleci/config.yml [hunk 2]
```diff
@@ -764,9 +764,42 @@ jobs:
...
+          command: sudo modprobe openvswitch
+      - run:
+          name: Setup Environment Variables
+          command: |
+            echo export MAGMA_ROOT=$(pwd) >> $BASH_ENV
+            echo export AWS_ACCESS_KEY_ID="$(printenv XWF_AWS_ACCESS_KEY_ID)" >> $BASH_ENV
+            echo export AWS_SECRET_ACCESS_KEY="$(printenv XWF_AWS_SECRET_ACCESS_KEY)" >> $BASH_ENV
+      - run:
+          name: "Set Python Version"
+          command: pyenv global 3.7.0
```
Comment (xjtian) [inline .circleci/config.yml]: is this actually necessary?

File: .circleci/config.yml [hunk 3]
```diff
@@ -764,9 +764,42 @@ jobs:
...
+      - run:
+          command: |
+            env
+            docker login -u ${XWF_ARTIFACTORY_USER} -p ${XWF_ARTIFACTORY_API_KEY} facebookconnectivity-openflow-xwfm-docker.jfrog.io
+            cd ${MAGMA_ROOT}/xwf/docker/
+            docker-compose build --parallel && docker-compose up -d && docker exec tests pytest --log-cli-level=info code/tests.py --type=analytic && docker-compose down
```
Comment (xjtian) [inline .circleci/config.yml]: we don't have to worry about the docker-compose down

**Review by xjtian — CHANGES_REQUESTED** [review]

**Comment by tmdzk** [comment]
https://app.circleci.com/pipelines/github/magma/magma/33/workflows/528e8200-22ed-426a-9267-fc6f1018bb58/jobs/411

Ready to be merged

**Review by xjtian — APPROVED** [review]

**Pull request merged**